app/assets/html/survey_questions_edit.html

```diff
@@ -0,0 +1,17 @@
+<div id="survey-questions-container">
+    <div class="header">{{ currentTopic }}</div>
+    <div class="spacing"></div>
+    <div id="topic-questions-container">
+        <div id="question{{$index+1}}" class="question-container" ng-repeat="question in questions">
```

Comment:
I will add this to pivotal as a task next iteration to change to using the same page, as that requires using state knowledge.